docs/docs/governance/unlock-dao-tokens.mdx

```diff
@@ -2,57 +2,34 @@
+The Unlock Protocol governance tokens, also known as UP tokens, are the governance tokens for the Unlock Protocol. Holders of UP tokens collectively form the [Unlock DAO](./unlock-dao).
 
-:::info
-UP (and UDT, its predecessor) are perfectly _optional_. Creators can deploy locks without knowledge of UP and their members can similarly purchase membership without knowing about these tokens.
-:::
+UP tokens were previously known as Unlock Discount Tokens (UDT).
 
-UDT tokens were initially created on Ethereum, and have also been bridged to multiple side-chains and networks such as Polygon, Gnosis Chain, Optimism, Arbitrum, Base and others.
+UDT tokens were initially created on Ethereum, and have been bridged to multiple side-chains and networks including Base, Polygon, Gnosis Chain, Optimism, Arbitrum and others.
 
-In July 2024, the DAO approved a migration to the Base network. In order to effectively vote on the Base network, a new token was introduced: UP (Unlock Protocol Token).
-UDT holders are invited to exchange their UDT for UP on the Base network. The migration is a 1:1000 swap and can be conveniently achieved through a [migration website](https://up-swap.unlock-protocol.com/) in 4 steps:
+In July 2024, the Unlock DAO approved a migration to the Base network. In order to effectively vote on the Base network, a new token was introduced: UP (Unlock Protocol Token).
+UDT holders can exchange their UDT for UP on the Base network. The migration is a 1-to-1000 swap and can be conveniently achieved through a [migration website](https://up-swap.unlock-protocol.com/) in 4 steps:
 
 - Bridging UDT to Base (from any network)
-- Delegeting votes on the Unlock DAO
+- Delegating votes on the Unlock DAO on Base
 - Approving tokens to be swapped
 - Swapping UDT for UP
 
 :::info
 The Unlock Foundation is offering a token incentive to perform the migration to UP. More details can be found on the [migration website](https://up-swap.unlock-protocol.com/).
 :::
 
-# Token Distribution
+## UP Token Contract Information
 
 The UP token has been deployed on Base at the following address: [0xac27fa800955849d6d17cc8952ba9dd6eaa66187](https://basescan.org/token/0xac27fa800955849d6d17cc8952ba9dd6eaa66187). It is using OpenZeppelin's ERC20 implementation and is an upgradable contract. The proxy admin is the Timelock contract on Base, which means that it can be upgraded by the Unlock DAO only.
 
-import { PieChart } from '@mui/x-charts/PieChart'
-
-<PieChart
-  series={[
-    {
-      data: [
-        {
-          value: 300000000,
-          label: 'Unlock Labs (locked)',
-        },
-        { value: 150000000, label: 'Community (float)' },
-        { value: 100000000, label: 'DAO' },
-        { value: 200000000, label: 'Liquidity Pool' },
-        { value: 250000000, label: 'Airdrops & Campaigns' },
-      ],
-    },
-  ]}
-  width={800}
-  height={300}
-/>
-
-# UP & UDT Addresses
+## UDT Contract Addresses
```

Comment:
Technically, the component below renders whatever governance token is set as the reward on per network. Right now, it is indeed UDT on all networks, but in the next few days/weeks, it will show UP for the Base network.

Maybe we keep this list and for each network we show both addresses (when applicable) UDT and UP, as well as which token is "rewarded". In that case, I will change the name of this section for something like "UDT and UP addresses per network" ? wdyt?